go-controller/pkg/ovn/obj_retry.go

```diff
@@ -91,34 +93,53 @@ func NewRetryObjs(
 	}
 }
 
-// addRetryObjWithAdd adds an object to be retried later for add
-func (r *retryObjs) addRetryObjWithAdd(obj interface{}) {
+// setRetryObjWithNoBackoff sets an object's backoff to be retried
+// immediately during the next retry iteration
+// Used only for testing right now
+func (r *retryObjs) setRetryObjWithNoBackoff(key string) {
+	r.retryMutex.Lock()
+	defer r.retryMutex.Unlock()
+	if entry, ok := r.entries[key]; ok {
+		entry.backoffSec = noBackoff
+		return
+	}
+
+	klog.Errorf("Unable to find entry with key %s", key)
+}
+
+// addRetryObjWithAddNoBackoff adds an object to be retried immediately for add
+func (r *retryObjs) addRetryObjWithAddNoBackoff(obj interface{}) {
```

Comment:
nit: this is too much adds in the naming for brain to parse. `add`RetryObj`WithAdd`NoBackoff not fault of this PR, I guess `WithAdd` implies add event..but now it implies `AddNoBackoff`

Reply:
i'm open to a better name, i just couldn't think of one